action.yml

```diff
@@ -11,8 +11,11 @@ inputs:
     description: 'Labels to be removed from an issue, seperated by commas.'
     required: false
   ignore-if-assigned:
-    description: "True/False value to indicate if no labels should be added or removed if there is an asignee to the issue"
+    description: "True/False value to indicate if no labels should be added or removed if there is an assignee to the issue."
     required: true
+  ignore-add-if-labeled:
```

Comment:
Open to naming / semantics here.  I thought `ignore-if-labeled` looked good because it parallels `ignore-if-assigned`, but not sure if it makes sense to ignore removing labels 🤔

Reply:
After sleeping on it, I arrived at the same conclusion.  People can use multiple actions / workflows for deeper logic.